beacon-chain/rpc/beacon/blocks.go

```diff
@@ -192,6 +194,24 @@ func (bs *Server) StreamBlocks(_ *ptypes.Empty, stream ethpb.BeaconChain_StreamB
 					// One nil block shouldn't stop the stream.
 					continue
 				}
+				if featureconfig.Get().NewStateMgmt {
+					hasStateSummaryDB := bs.BeaconDB.HasStateSummary(bs.Ctx, bytesutil.ToBytes32(data.SignedBlock.Block.ParentRoot))
+					hasStateSummaryCache := bs.stateSummaryCache.Has(bytesutil.ToBytes32(blk.Block.ParentRoot))
```

Comment:
You dont need state summary cache. It's only for syncing

Reply:
it didn't work in runtime without it

Reply:
i tried

Reply:
```[2020-06-15 17:59:00]  WARN rpc: No access to parent state to verify block signature blockSlot=10108
[2020-06-15 17:59:00]  INFO blockchain: Synced new block block=0xa8ba7d59... epoch=315 finalizedEpoch=313 finalizedRoot=0x9a4a86ae... slot=10108
[2020-06-15 17:59:00]  INFO blockchain: Finished applying state transition attestations=44 attesterSlashings=0 deposits=0 proposerSlashings=0 voluntaryExits=0```

Reply:
Ah ok. I'll make improvement on not making state summary cache a criterial. For now, this is good 👍